### ISSUE TITLE
Separating left/right turns and spatial locations

### ISSUE THREAD

**Issue by jtmiles** [Humphries-Lab/Bayesian_Strategy_Analysis_MATLAB#1]
Hi! Just diving into this package and very excited to give it a try, but would like your input on how to adapt it to our behavioral paradigm.

Looking over how to re-format my behavior data tables, I see that values in the "Choice" column are expected to be "left" or "right" for certain strategy models. On our plus maze, though, trials can start in either the "North" or "South" arms, and require a choice of either "East" or "West" reward sites. This, of course, means that left/right turns lead to different locations on the maze depending on where the trial starts.

My understanding is that this will not work quite right with spatial strategy models that rely on the "Choice" column to compute the current strategy. Is a good solution to include a new column that tells which location (East/West) was chosen, and write some new strategy models that parse that column? Or perhaps you have other suggestions?

**tl;dr: How do you recommend disambiguating left/right turns from turns toward a specific location when going left or right do not always lead to the same place?**

Thanks!
Jesse Miles
Ph.D. candidate
Mizumori & Gire labs
Psychology Dept.
University of Washington

**Comment by mdhumphries** [comment]
Hi Jesse, thanks for trying out the toolbox

I think you're right that the strategy models we've included don't cover your case. We've egocentric (left/right) and proximal-cue based allocentric strategies, but not distal-cue "location"-based allocentric strategies.

So I think your diagnosis is correct, and your simple additions should work: 
- include a column for location (e.g. Location, with variables "east" and "west") in your CSV file (or whatever you're using)
- create two new strategy models (e.g. "go_east" and "go_west") based on e.g. "go_left": the logic is the same, just a change of variable names in the "if" statement (e.g. if trial_data.Location(end) == "east" for go_east). 

Then you can contrast the probabilities of the egocentric (go_left, go_right) and allocentric (go_east, go_west) strategies

Cheers
Mark

**Comment by jtmiles** [comment]
Great! Thanks for your prompt reply. I'll consider this issue closed and give this a try soon
